Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -11,13 +11,22 @@ VENDOR_DIR="$HOMEBREW_LIBRARY/Homebrew/vendor"
 # Built from https://github.com/Homebrew/homebrew-portable.
 if [[ -n "$HOMEBREW_MACOS" ]]
 then
-  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  # Universal 10.4 build
+  if [[ "$HOMEBREW_MACOS_VERSION_NUMERIC" -lt "100500" || "$HOMEBREW_PROCESSOR" != "Intel" ]]
   then
+    ruby_URL="https://archive.org/download/tigerbrew/portable-ruby-2.2.7.tiger.bottle.tar.gz"
```

Comment:
2.0 has a bug which prevents it from being built as a universal binary, and 2.4 has a bug which prevents it from being built statically.